src/qml/editorwidgets/CheckBox.qml

```diff
@@ -8,6 +8,9 @@ import "."
 EditorWidgetBase {
   height: childrenRect.height
 
+  property string checkedLabel: config['TextDisplayMethod'] === 1 && config['CheckedState'] != '' ? config['CheckedState'] : qsTr('True')
+  property string uncheckedLabel: config['TextDisplayMethod'] === 1 && config['UncheckedState'] != '' ? config['UncheckedState'] : qsTr('False')
```

Comment:
```suggestion
  property string checkedLabel: config['TextDisplayMethod'] === 1 && config['CheckedState'] !== '' ? config['CheckedState'] : qsTr('True')
  property string uncheckedLabel: config['TextDisplayMethod'] === 1 && config['UncheckedState'] !== '' ? config['UncheckedState'] : qsTr('False')
```
  Is it guaranteed `config['CheckedState']` is always a string, but not `undefined` or `null`? Asking because `undefined != ''` (note the single `=` sign).

Reply:
@suricactus , I expect it can be undefined / null, which I thought meant the != was properly used here (i.e., if it's != '' or != undefined/null) -- or did I get those === !== wrong?

Reply:
Hope this one helps what I meant: https://jsbin.com/wicuzudoqu/edit?js,console